### DISCUSSION TITLE
How to set up `www` prefix (with Google Domains)?

### DISCUSSION THREAD

**Discussion by piet-maier** [vercel/community#1630] [Help] [answered]
### Summary

I have a project deployed on Vercel, and a domain that I bought from Google Domains. For this question, let's say my domain is `domain.page`. To connect the domain to my Vercel deployment, I followed a guide I found online. These are my current DNS settings on Google:

<img width="705" alt="Screenshot 2023-02-26 at 16 57 14" src="https://user-images.githubusercontent.com/93382331/221421665-8eb92d65-8800-4137-bb6c-ef0f275cec4d.png">

Using `domain.page` to get to my page works, but using `www.domain.page` does not. I've never done something like this before, so I don't understand why and how to change that. Can somebody help?

**Comment by piet-maier** [comment, marked as answer]
Okay, I noticed that I didn't have the redirect on the Vercel domain dashboard, added that, changed to custom name server on Google and set Vercel's, and that solved it.

**Reply by amyegan** [reply]
Thanks for coming back to share this! I'm glad you found a solution that worked for your project, but I also want to share a few links here for anyone else who comes across this question in the future.

- [Adding & Configuring a Custom Domain](https://vercel.com/docs/concepts/projects/domains/add-a-domain#)
- [Deploying & Redirecting Domains](https://vercel.com/docs/concepts/projects/domains/deploying-and-redirecting#)
- [Assigning a Custom Domain](https://vercel.com/docs/concepts/get-started/assign-domain#)
- [Why must we use the Domain Nameservers method for Wildcard Domains on Vercel?](https://vercel.com/guides/why-use-domain-nameservers-method-wildcard-domains)
